### DISCUSSION TITLE
2022-10-05 Weekly Meeting 76 Agenda

### DISCUSSION THREAD

**Discussion by antpb** [omigroup/omigroup#300] [Weekly Meetings]
Weekly Meeting 76 is happening on Wednesday October 5, 2022 from 9:00 AM PST(16:00 UTC) - 10:00 AM PST (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat. 

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda
- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers

### Working Group Updates:
- [Scripting group](https://github.com/omigroup/omi-scripting-group)
  - OMI developer hour* 
- [glTF Extensions Group](https://github.com/omigroup/gltf-extensions)
  - Progress on OMI_ref and authoring extensions
- [Media Group](https://github.com/omigroup/media)
- [OMI Website update](https://github.com/omigroup/website)
- [MSF delegates](https://github.com/omigroup/MSF-Delegates) (setting up the repo)
- [M3 avatar interop](https://github.com/M3-org/avatar-interop#calendar)
- [UX Research: TBD](https://github.com/omigroup/ux-research) (setting up the repo) 
  - #222 repo is formed, when do we have the first meeting? https://github.com/omigroup/ux-research/issues/6
- any others?

### Discussions: 
- Discuss Lamina1 Open Metaverse Conference Partnership. 

Core issue: OMI is listed as "Official Partner" without group vote on participating in the Lamina1 event. There has been press listing OMI as an Official Partner so discussion is needed around how we move forward. A hackmd was drafted by antpb and jin outlining options to move forward: https://hackmd.io/7fXyAOcKTGG9dEKvSgp-ag

### Future demo themes: (Vote now in the `#omi-vote` Discord channel!)
- `OMI-audio-emitter` exploration and demo 
- `OMI-collider` exploration and demo 
- `Generative and AI tooling`
- Any other suggestions?

**Comment by mrmetaverse** [comment]
# UX Group 

- recent announcement on twitter https://twitter.com/mrmetaverse/status/1576999682374782976 
- Team is looking to have first meeting to kick off the repo 
- first challenge: Portals? https://github.com/omigroup/ux-research/issues/8 | https://github.com/omigroup/ux-research/milestone/2
- get involved here https://github.com/omigroup/ux-research

**Reply by mrmetaverse** [reply]
re: portals: 
design guidelines, not standards. recommendations of best practices.

**Comment by mrmetaverse** [comment]
# Open Metaverse Conference 
- Jamil with Lamina1 joined us today to talk about their intentions for the conference. 
- We will review our group notes and requests. 
- How does OMI want to show up? 
- Do we need to make any changes? Are there any risks associated?

**Reply by mrmetaverse** [reply]
Reason to doing the event: 
- the ideals of the Open Metaverse are essential to why Lamina1 exists.

**Reply by mrmetaverse** [reply]
Imagine how Microsoft, or Unreal, or Unity would sponsor the Game Developers Conference (GDC). We intend to make this editorially driven in that way. Above board with clearly defined sessions and hopefully a collaborative approach to the conference. 

Have we made any missteps and how can we work together?

**Reply by mrmetaverse** [reply]
The hackmd with thoughts and questions: https://hackmd.io/7fXyAOcKTGG9dEKvSgp-ag?both

**Reply by mrmetaverse** [reply]
It's not exactly best practice to put out a release on a Friday, we wanted to get a save the date out. The term used "official partners," refers to standards groups that we admire and already believe match the spirit of what we see for the event, i.e. being able to ask all questions where nothing is really off limits. If you are interested in the open Metaverse, that's the premise.

It's safe to ask "why even blockchain?"

**Reply by mrmetaverse** [reply]
Q: What does it mean to be a partner? 
We want to provide a platform for your views. The spirit of partnership is an invite to present a panel, free passes for your members, the ability to participate in networking, help shape what the event is based on your own goals and interests. Fundamentally, a seat at the table to help advise overall what the conference should be at it's early stages. 

We have reached out to a few folks we want to have high-level talks, keynotes, and fireside chats, but our goal is to involved groups like OMI to shape the event.

**Reply by mrmetaverse** [reply]
Q: Lamina1 has obviously been spoken about in our community before. We have a diverse community, with diverse levels of interest and experience in blockchain (i.e. many still strongly dislike it). It feels like LeapCon, which was a great experience. OMI is a Metaverse project, and many of our community members are anonymous, we need to have a strong metaverse element for this conference. How can we incorporate a virtual element that doesn't suck? Something more like a virtual burn (Burning Man)

**Reply by mrmetaverse** [reply]
Criticisms: 
- Many of us obviously respect Neil Stephenson. It feels like the carbon credits aren't addressing a real problem though. LIke more noise and talk, and it rubs people the wrong way. 
- Why another Layer 1 chain? We know VCs love L1 hype, but why do we actually need _another_ potential rug pull? If we hadn't seen this hundreds of times before, we would be more open. So at this point a lot of the community is a bit cynical about it. 
- Of course we are all asking ourselves if we just bite the bullet and get involved and make an impact. Ideally, the token is entirely dedicated to building the open Metaverse. 
- OMA3 - They are doing cool things, but they haven't engaged with OMI before, and we don't all understand the need for a "blockchain" alternative to OMI, when we are all building towards the same Metaverse.

**Reply by mrmetaverse** [reply]
JA: Can we also place emphasis on a more diverse panel?

**Reply by mrmetaverse** [reply]
Answer: We don't want this to be another talky thing. We intend to have a full demo room. We want to host a dozen or so extraordinary technologies that show, and don't just tell, what the Open Metaverse can be. 

We also want to help those who can't afford a sponsorship. We are interested in a large deep pocketed donor to help us subsidize speaking - ideally the ability to speak for free. 

We want people to be able to show what they are doing without having to pay for it.

**Reply by mrmetaverse** [reply]
Answer: Why another L1 chain? 

- that was my first questions as well, but have learned enough about what is going on to feel more comfortable about it. 
- Specifically having a storage layer on the block chain, making wallets more accessible, and overall the creator community would be purpose-built for the Metaverse, specifically tuned to your interests.
